### DISCUSSION TITLE
Star my repository PLS i'll give it back :D //help with stars

### DISCUSSION THREAD

**Discussion by ogprzemo** [drknzz/GitHub-Achievements#243] [Q&A]
https://github.com/Przems0n711/AuthCracker

thank you

**Comment by EliasAfara** [comment]
sure

**Comment by Nielioo** [comment]
Already star yours, give me my repo star too please..
https://github.com/Nielioo/Nielioo

**Reply by Mattral** [reply]
Done Please star this too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by Mattral** [comment]
Done! please star this too!

https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by EliasAfara** [reply]
Do you want to keep your Github contribution steak alive without making commits personally ?
Check out my new project [contribution-streak-maintainer](https://www.npmjs.com/package/contribution-streak-maintainer), it's an npm package, you only need to create a yml file and generate a Github token, very simple procedure.

**Comment by the1Riddle** [comment]
you have to get 16 stars to get the star achievement badge on your profile.

**Comment by ganochenkodg** [comment]
@Przems0n711 done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Reply by the1Riddle** [reply]
@Przems0n711 done 

checkout this #274
